src/Types/Uint/Uint.ts

```
@@ -0,0 +1,358 @@
+ * @param value - The Uint value.
+ * @param bits - The bit size of the integer.
+ */
+export const toBytesArray = <BITS extends number>(
+  value: Uint<BITS>,
+  bits: BITS,
+): Effect.Effect<Uint8Array<ArrayBufferLike>> =>
+  Effect.sync(() => {
+    const bytes = Math.ceil(bits / 8);
+    const hex = Hex.padLeft(`0x${value.toString(16)}`, bytes * 2);
+    return new Uint8Array(
+      hex.slice(2).match(/.{1,2}/g)?.map(byte => parseInt(byte, 16)) || []
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)